content/200-concepts/100-components/01-prisma-schema/09-index-configuration.mdx

```diff
@@ -48,8 +58,8 @@ model Id {
 But this is not not valid in MySQL because it exceeds MySQL's index storage limit and therefore Prisma rejects the data model. The generated SQL would be rejected by the database.
```

Comment:
```suggestion
But this is not valid in MySQL because it exceeds MySQL's index storage limit and therefore Prisma rejects the data model. The generated SQL would be rejected by the database.
```